hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/AWSDmsTransformer.java

```diff
@@ -19,8 +19,8 @@
 package org.apache.hudi.utilities.transform;
 
 import org.apache.hudi.common.config.TypedProperties;
-import org.apache.hudi.common.model.AWSDmsAvroPayload;
 import org.apache.hudi.common.util.Option;
+import org.apache.hudi.payload.AWSDmsAvroPayload;
 
 import org.apache.spark.api.java.JavaSparkContext;
```

Comment:
why this change? the old class is not meant for bwc where users are loading the class via FQCN. internal codebase should always use the new class.

Reply:
fixed